src/msg_handler.c

```diff
@@ -53,6 +53,11 @@ void handle_msg(char* buf, char* response, int max_len) {
         if(next_arg == NULL) {return;}
 
         if (strcmp(next_arg, "SET") == 0) {
+            // inhibit setting brain port
+            if (output_num == BRAIN_OUTPUT) {
+                append_str(response, "NACK:Brain output cannot be controlled", max_len);
```

Comment:
Nit: This message isn't quite true. It _can_ be controlled (using the `BRAIN` command), just not with this command.

Reply:
The `*SYS` commands are only intended for internal use so this is valid from an end-user's perspective.